Core/FieldHandler/EzCountry.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Kaliop\eZMigrationBundle\Core\FieldHandler;
+
+use eZ\Publish\API\Repository\ContentTypeService;
+use eZ\Publish\Core\FieldType\Country\Value as CountryValue;
+use eZ\Publish\Core\FieldType\Selection\Value as SelectionValue;
+use Kaliop\eZMigrationBundle\API\FieldValueImporterInterface;
+
+class EzCountry extends AbstractFieldHandler implements FieldValueImporterInterface
+{
+    private $contentTypeService;
+
+    private $countries;
+
+    public function __construct(ContentTypeService $contentTypeService, array $countries)
+    {
+        $this->contentTypeService = $contentTypeService;
+        $this->countries = $countries;
+    }
+
+    /**
+     * Creates a value object to use as the field value when setting a country field type.
+     *
+     * @param $fieldValue string|string[] should work
+     * @param array $context The context for execution of the current migrations. Contains f.e. the path to the migration
+     * @return CountryValue
+     */
+    public function hashToFieldValue($fieldValue, array $context = array())
+    {
+        if ($fieldValue === null) {
+            return new CountryValue();
+        }
+
+        $fieldSettings = $this->loadFieldSettings($context['contentTypeIdentifier'], $context['fieldIdentifier']);
+
+        $countries = [];
+        if (!$fieldSettings['isMultiple'] && is_string($fieldValue)) {
+            $countries = [$fieldValue => $this->countries[$fieldValue]];
+            return new CountryValue($countries);
+        }
+
+        if (is_string($fieldValue)) {
+            $fieldValue = array($fieldValue);
+        }
+
+        foreach ($fieldValue as $countryCode) {
+            $countries[$countryCode] = $this->countries[$countryCode];
```

Comment:
q: what happens if `$this->countries[$countryCode]` is not set? shall we throw, or will the kernel do it for us ?

Reply:
kernel will throw. throwing here could be an option too.